.github/workflows/ci.yml

```
@@ -79,7 +79,7 @@ jobs:
         run: pipenv graph
 
       - name: Run complete test suite
-        run: pipenv run py.test --flake8 -s -vvv africanus/
+        run: NUMBA_NRT_STATS=1 pipenv run py.test --flake8 -s -vvv africanus/
```

I don't think `NUMBA_NRT_STATS=1` gets inherited by the  virtual environment created by pipenv


```suggestion
        run: pipenv run NUMBA_NRT_STATS=1 py.test --flake8 -s -vvv africanus/
```

In a fresh python 3.9 virtual environment the test suite is running:

```bash
$ NUMBA_NRT_STATS=1 py.test --flake8 -s -vvv africanus/
```